config/zones.json

```diff
@@ -5,11 +5,13 @@
       [2.2650907800001505, 43.14936167400003]
     ],
     "capacity": {
-      "hydro": 45,
-      "nuclear": 0
+      "biomass": 5,
+      "gas": 1.7,
+      "hydro": 46,
+      "solar": 0.231
```

Comment:
When reviewing #4321 I noticed IRENA lists 4 MW of installed solar, if you added that it would be great.

Reply:
Does FEDA include residential and commercial rooftop solar?

I think most if not all of our other sources do that so if they don't it might be better to go with IRENA.

As for when there is conflicting information we try and go with the official ones but if there is another source with more comprehensive data (better breakdown per mode for example) that can be used as a complement or even fully.

I don't think we have had two fully conflicting (official) sources yet so not sure what we'd do if we ran into that problem.